_partials/_devops-cli-get-started.md

```diff
@@ -0,0 +1,217 @@
+
+<Procedure>
+
+1. ** Install $CLI_LONG**
+
+   Use the Terminal to install the $CLI_SHORT: 
+   <Tabs label="Install Tiger CLI" persistKey="os">
+
+    <Tab title="Debian" label="debian">
+
+    ```shell
+    curl -s https://packagecloud.io/install/repositories/timescale/tiger-cli/script.deb.sh | sudo bash
+    sudo apt-get install tiger-cli
+    ```
+    
+    </Tab>
+    
+    <Tab title="Ubuntu" label="ubuntu">
+
+    ```shell
+    curl -s https://packagecloud.io/install/repositories/timescale/tiger-cli/script.deb.sh | sudo bash
+    sudo apt-get install tiger-cli
+    ```
+    </Tab>
+    
+    <Tab title="Red Hat" label="redhat">
+   
+    ```shell
+    curl -s https://packagecloud.io/install/repositories/timescale/tiger-cli/script.rpm.sh | sudo bash
+    sudo yum install tiger-cli
+    ```
+   
+    </Tab>
+    
+    <Tab title="Fedora" label="fedora">
+
+    ```shell
+    curl -s https://packagecloud.io/install/repositories/timescale/tiger-cli/script.rpm.sh | sudo bash
+    sudo yum install tiger-cli
+    ```
+    
+    </Tab>
+
+    <Tab title="MacOs" label="macos">
+
+    ```shell
+    brew install --cask timescale/tap/tiger-cli
+    ```
+
+    </Tab>
+
+    <Tab title="x-platform" label="xplatform">
+
+    ```shell
+    curl -fsSL https://tiger-cli-releases.s3.amazonaws.com/install/install.sh | sh
+    ```
+
+    </Tab>
+
+    </Tabs>
+ 
+1. **Set up API credentials**
+
+   1. Log $CLI_LONG into your $CLOUD_LONG account
+ 
+      ```shell
+      tiger auth login
+      ```
+      $CLI_LONG opens $CONSOLE_SHORT in your browser. Login, then click `Authorize`.  
+
+   1. Select a $PROJECT_LONG. 
+
+      $CLI_LONG stores your authentication information locally in `~/.config/tiger/config.yaml`.
+    
+1. **Test your authenticated connection to $CLOUD_LONG by listing services**
+
+    ```bash
+    tiger service list
+    ```
+
+   This call returns something like:
+    - No services:
+      ```terminaloutput
+      🏜️  No services found! Your project is looking a bit empty.
+      🚀 Ready to get started? Create your first service with: tiger service create
+      ```
+    - One or more services:
+
+      ```terminaloutput
+      ┌────────────┬─────────────────────┬────────┬─────────────┬──────────────┬──────────────────┐
+      │ SERVICE ID │        NAME         │ STATUS │    TYPE     │    REGION    │     CREATED      │
+      ├────────────┼─────────────────────┼────────┼─────────────┼──────────────┼──────────────────┤
+      │ tgrservice │ tiger-agent-service │ READY  │ TIMESCALEDB │ eu-central-1 │ 2025-09-25 16:09 │
+      └────────────┴─────────────────────┴────────┴─────────────┴──────────────┴──────────────────┘
+      ```
+
+</Procedure>
+
+
+## Create your first service
+
+Create a new $SERVICE_LONG using $CLI_LONG with a secure configuration:
+
+<Procedure>
+
+1. **Submit a service creation request**
+   ```shell
+   tiger service create
+   ```
+   $CLOUD_LONG creates a `#dev` environment for you. You see something like:
```

Comment:
What does "creates a `#dev` environment for you" mean? Just that it creates a service that's tagged as a dev service? The wording seems a little confusing to me.